src/features/orientation/components/selected-filters/selected-filters.component.html

```diff
@@ -1,44 +1,53 @@
-<ng-container *ngIf="filterPresentation as filter">
-  <div class="form-check form-check-inline form-chip mb-3" *ngIf="filter.services">
-    <button class="chip-button" (click)="resetForm(filter.services, 'services')">
-      {{ filter.services }}
+<ng-container *ngIf="filterPresentation as filter" style="flex-wrap: wrap" class="d-flex">
+  <div class="form-check form-check-inline form-chip mb-1" *ngIf="filter.services">
+    <button class="chip-button py-1 fw-bold" style="max-width: 250px" (click)="resetForm(filter.services, 'services')">
+      {{ filter.services.length > 18 ? filter.services.substring(0, 18) + '...' : filter.services }}
```

Comment:
```suggestion
  <div class="form-check form-check-inline form-chip mb-1" *ngIf="filter.services">
    <button class="chip-button py-1 fw-bold" (click)="resetForm(filter.services, 'services')">
     filter.services
```
Affichage du texte trop long géré en css